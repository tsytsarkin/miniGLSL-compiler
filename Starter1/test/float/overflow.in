32766.9
32767.01
