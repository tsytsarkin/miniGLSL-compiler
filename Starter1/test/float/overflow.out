TOKEN 258 : 32766.9

LEXICAL ERROR, LINE 2: Float out of bounds
